### PR TITLE
week1 react homework

### PR DESCRIPTION
**Pull request by krithi-kondana**

### CODE REVIEW

File: react/week1/todos/src/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+// import * as serviceWorker from './serviceWorker';
+
+
+const root = document.getElementById('root')
+
+ function TodoItem(props) {
+   return <li>{props.description}, {props.deadline}</li>
+ }
+
+class Todo extends React.Component {
```
Comment (fernandoabolafio) [inline react/week1/todos/src/index.js]: You could use a better naming here such as TodoList. It's always nice to look at the component and to know immediately what it is doing.

File: react/week1/todos/src/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  render() {
+   const todoList = this.props.todos;
+   const result = todoList.map(element => {
+    return <TodoItem description={element.description} deadline={element.deadline}></TodoItem>
```
Comment (fernandoabolafio) [inline react/week1/todos/src/index.js]: 👍

File: react/week1/todos/src/index.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+const todosArray = [
+  { description: 'Watch TV', deadline: new Date('2020-04-24').toDateString()},
+  { description: 'Listen music', deadline: new Date('2020-02-20').toDateString()},
+  { description: 'Wash clothes', deadline: new Date('2020-03-28').toDateString()}
+]
+
+class Header extends React.Component {
```
Comment (fernandoabolafio) [inline react/week1/todos/src/index.js]: Nothing is wrong here, but simple components such as this one can be written just as plain functions:

```jsx
function Header() {
 return (<header>
      <h3>Todo list</h3>
    </header>))
}
```

Less verbosity ;)

File: react/week1/todos/src/index.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      </footer>
+    )
+  }
+}
+
+ReactDOM.render(
+  <div id="todo">
+    <Header></Header>
+    <Todo todos={todosArray}></Todo>
+    <Footer></Footer>
```
Comment (fernandoabolafio) [inline react/week1/todos/src/index.js]: When you're not passing anything inside the component tags you can just write: 
```suggestion
    <Header />
    <Todo todos={todosArray} />
    <Footer />
```

File: react/week1/todos/src/index.js (hunk 2 above)
Comment (krithi-kondana) [reply react/week1/todos/src/index.js]: Thanks for the feedback @fernandoabolafio I have made necessary changes.

**Pull request merged**